FogCreek/Glitch-Community
# Superb guide: Move ssr build into webpack for faster server startup

## Links
https://superb-guide.glitch.me/

## Changes:
- Get rid of the separate `babel` call, now ssr is prepared as part of the webpack build
  - Less confusing output: there's a single 'build complete' message
  - The server only reloads once the entire build is done, not every time babel writes out a single file
- Don't watch for file changes when `BUILD_TYPE=static`, so chokidar and require.cache manipulation don't even come into play

## How To Test:
Try running the site with `BUILD_TYPE` set to `memory` `watcher` and `static`. In watcher mode it should only ssr load once on startup and again after the build finishes, rather than on every request until babel finishes.
